xblock/mixins.py

```diff
@@ -516,6 +516,10 @@ def _add_field(self, node, field_name, field):
         as an xml attribute or creates a separate child node.
         """
         value = field.to_string(field.read_from(self))
+
+        if not isinstance(value, basestring):
```

Comment:
@mtyaka 

Changing `to_json` - yes, that's an alternative way to do that (and I've tried that and it works), but `None` is pretty much valid result for JSON. On the other hand, the issue is only observed with export, so this fix have a narrower impact by only fixing the behavior where it is broken.

`if value is None` - I thiought `basestring` check is more robust as `node.set` expects string, but on the second thought it appears as too broad patch, i.e. it might mask other issues (in case other non-string value is returned). so I think I'lll change it to `if value is None`